.eslintrc.js

```diff
@@ -5,6 +5,16 @@ module.exports = {
     // "jsx-curly-spacing": [2, "always", {"alternative": true}], // https://github.com/yannickcr/eslint-plugin-react/issues/388
     'react/jsx-curly-spacing': [1, 'always'],
     'react/prop-types': [0],
+    'react/sort-comp': [1, {
```

Comment:
I don't mind putting the nine components that have sort order lint errors in the default sort order. It would have some awkward effects though, particularly it would result in the `classes` method from `ReactCSS` being counted as a user-defined method (and thus placed in between the life cycle methods and `render`). I will just take out the custom sorting rule for now.